Failed to install Error: Command failed with exit code 1

Hello the plugin installation is done without problem, once I run ionic cordova prepare android I have this error
(Sorry for my english i use google translate)

```
Running hook: C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina\plugins\cordova-plugin-fcm-with-dependecy-updated\scripts\install_ionic_dependencies.bat C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'install', '--loglevel', 'error', '--no-progress' ]
}
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'install', '--loglevel', 'error', '--no-progress' ]
}
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'install', '--loglevel', 'error', '--no-progress' ]
}
Failed to install 'cordova-plugin-fcm-with-dependecy-updated': Error: Command failed with exit code 1: C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina\plugins\cordova-plugin-fcm-with-dependecy-updated\scripts\install_ionic_dependencies.bat C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina
    at makeError (C:\Users\alexf\AppData\Roaming\npm\node_modules\cordova\node_modules\execa\lib\error.js:59:11)
    at handlePromise (C:\Users\alexf\AppData\Roaming\npm\node_modules\cordova\node_modules\execa\index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
Command failed with exit code 1: C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina\plugins\cordova-plugin-fcm-with-dependecy-updated\scripts\install_ionic_dependencies.bat C:\Users\alexf\Desktop\Ionic\091120\2\LaPamparina
[ERROR] An error occurred while running subprocess cordova.

        cordova.cmd platform add android --save exited with exit code 1.
```

**Package.json**
```json
{
  "name": "LaPamparina",
  "version": "0.0.1",
  "author": "Ionic Framework",
  "homepage": "https://ionicframework.com/",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/common": "^10.2.2",
    "@angular/core": "^10.2.2",
    "@angular/fire": "^6.0.4",
    "@angular/forms": "^10.2.2",
    "@angular/platform-browser": "^10.2.2",
    "@angular/platform-browser-dynamic": "^10.2.2",
    "@angular/router": "^10.2.2",
    "@ionic-native/admob-free": "^5.29.0",
    "@ionic-native/core": "^5.0.0",
    "@ionic-native/firebase": "^5.29.0",
    "@ionic-native/splash-screen": "^5.0.0",
    "@ionic-native/status-bar": "^5.0.0",
    "@ionic/angular": "^5.0.0",
    "@ionic/lab": "^3.2.9",
    "cordova-res": "^0.15.2",
    "firebase": "^8.0.1",
    "rxjs": "^6.6.3",
    "tslib": "^2.0.0",
    "zone.js": "^0.11.3"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^0.1002.0",
    "@angular/cli": "^10.2.0",
    "@angular/compiler": "^10.2.2",
    "@angular/compiler-cli": "^10.2.2",
    "@angular/language-service": "^10.2.2",
    "@ionic/angular-toolkit": "^2.3.0",
    "@types/jasmine": "^3.6.1",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^14.14.6",
    "codelyzer": "^6.0.0",
    "cordova-admob-sdk": "^0.24.1",
    "cordova-plugin-admob-free": "^0.27.0",
    "cordova-plugin-device": "^2.0.2",
    "cordova-plugin-fcm-with-dependecy-updated": "^7.6.0",
    "cordova-plugin-ionic-keyboard": "^2.2.0",
    "cordova-plugin-ionic-webview": "^4.2.1",
    "cordova-plugin-splashscreen": "^5.0.2",
    "cordova-plugin-statusbar": "^2.4.2",
    "cordova-plugin-whitelist": "^1.3.3",
    "cordova-promise-polyfill": "0.0.2",
    "jasmine-core": "^3.6.0",
    "jasmine-spec-reporter": "^6.0.0",
    "karma": "^5.2.3",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~3.0.2",
    "karma-jasmine": "^4.0.1",
    "karma-jasmine-html-reporter": "^1.5.0",
    "protractor": "~7.0.0",
    "ts-node": "^9.0.0",
    "tslint": "~6.1.0",
    "typescript": "^4.0.5"
  },
  "description": "An Ionic project",
  "cordova": {
    "plugins": {
      "cordova-plugin-admob-free": {
        "ADMOB_APP_ID": "ca-app-pub-xxxx~xxxxx"
      },
      "cordova-plugin-whitelist": {},
      "cordova-plugin-statusbar": {},
      "cordova-plugin-device": {},
      "cordova-plugin-splashscreen": {},
      "cordova-plugin-ionic-webview": {
        "ANDROID_SUPPORT_ANNOTATIONS_VERSION": "27.+"
      },
      "cordova-plugin-ionic-keyboard": {},
      "cordova-plugin-fcm-with-dependecy-updated": {
        "ANDROID_DEFAULT_NOTIFICATION_ICON": "@mipmap/ic_launcher",
        "ANDROID_FIREBASE_BOM_VERSION": "26.0.0",
        "ANDROID_GOOGLE_SERVICES_VERSION": "4.3.4",
        "ANDROID_GRADLE_TOOLS_VERSION": "4.1.0",
        "IOS_FIREBASE_MESSAGING_VERSION": "7.0.0"
      }
    },
    "platforms": []
  }
}
```

Yes, broken for me too.

```Installing "cordova-plugin-fcm-with-dependecy-updated" for android
Subproject Path: CordovaLib
Subproject Path: app
Running hook: D:\Projects\unitaskr\ionic-new\myApp\plugins\cordova-plugin-fcm-with-dependecy-updated\scripts\install_ionic_dependencies.bat D:\Projects\unitaskr\ionic-new\myApp
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    [... lines matching original stack trace ...]
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    [... lines matching original stack trace ...]
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
    at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    [... lines matching original stack trace ...]
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Failed to install 'cordova-plugin-fcm-with-dependecy-updated': Error: Command failed with exit code 1: D:\Projects\unitaskr\ioionic-new\myApp\plugins\cordova-plugin-fcm-with-dependecy-updated\scripts\install_ionic_dependencies.bat D:\Projects\unitaskior\ionic-new\myApp
    at makeError (C:\Users\craig\AppData\Roaming\npm\node_modules\cordova\node_modules\execa\lib\error.js:59:11)
    at handlePromise (C:\Users\craig\AppData\Roaming\npm\node_modules\cordova\node_modules\execa\index.js:114:26)
Command failed with exit code 1: D:\Projects\unitaskr\ionic-new\myApp\plugins\cordova-plugin-fcm-with-dependecy-updated\scriptpts\install_ionic_dependencies.bat D:\Projects\unitaskr\ionic-new\myApp
    at process._tickCallback (internal/process/next_tick.js:68:7)
[ERROR] An error occurred while running subprocess cordova.
```
```
PS> ionic info

Ionic:

   Ionic CLI                     : 6.12.1 (C:\Users\craig\AppData\Roaming\npm\node_modules\@ionic\cli)
   Ionic Framework               : @ionic/angular 5.4.4
   @angular-devkit/build-angular : 0.1000.8
   @angular-devkit/schematics    : 10.0.8
   @angular/cli                  : 10.0.8
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : android 9.0.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 10 other plugins)      

Utility:

   cordova-res (update available: 0.15.2) : 0.15.1
   native-run                             : 1.2.2

System:

   Android SDK Tools : 26.1.1 (C:\Users\craig\AppData\Local\Android\Sdk)
   NodeJS            : v10.23.0 (D:\Program Files (x86)\node-v10.23.0-win-x64\node.exe)
   npm               : 6.9.2
   OS                : Windows 10
```

The path is well available in the environment variables. Now i have this error

`FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:debugCompileClasspath'.
   > Could not find com.google.firebase:firebase-bom:null.
     Required by:
         project :app
   > Could not find com.google.firebase:firebase-messaging:.
     Required by:
         project :app

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 20s
Command failed with exit code 1: C:\Users\alexf\Desktop\Ionic\191120\LaPamparina\platforms\android\gradlew cdvBuildDebug -b C:\Users\alexf\Desktop\Ionic\191120\LaPamparina\platforms\android\build.gradle
[ERROR] An error occurred while running subprocess cordova.

        cordova.cmd build android exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.`

--- a/scripts/install_ionic_dependencies.js
+++ b/scripts/install_ionic_dependencies.js
@@ -45,8 +45,9 @@ const installIonicDependencies = function () {
         return;
     }
 
+    const npm = process.platform === 'win32' ? 'npm.cmd' : 'npm';
     helpers
-        .execute('npm', ['install', '--loglevel', 'error', '--no-progress'])
+        .execute(npm, ['install', '--loglevel', 'error', '--no-progress'])
         .catch(function (e) {
             helpers.logError('Failed to auto install Ionic dependencies!', e);
             helpers.logError(

